tgs/src/test/cpp/tgs/BigContainers/BasicBloomFilterTest.cpp

```diff
@@ -63,12 +63,12 @@ class BasicBloomFilterTest : public CppUnit::TestFixture
         boost::bloom_filters::boost_hash<size_t, 0x7545E14679E2A9E3>,
         boost::bloom_filters::boost_hash<size_t, 0x5BD062C2515F007C>,
         boost::bloom_filters::boost_hash<size_t, 0x4DB127F812200854> > > Bloom;
-    Bloom* b = new Bloom();
-    Bloom& uut = *b;
+    //  NOTE: Bloom must be created on the heap and not the stack, thus the "new Bloom()"
+    shared_ptr<Bloom> b(new Bloom());
```

Comment:
Very important note to create the Bloom on the heap but we can't just leak the memory either, can we?